Tribler/Core/Modules/restapi/downloads_endpoint.py

```diff
@@ -134,6 +137,35 @@ def __init__(self, session, infohash):
         self.infohash = bytes(infohash.decode('hex'))
         self.putChild("torrent", DownloadExportTorrentEndpoint(session, self.infohash))
 
+    @staticmethod
+    def create_dconfig_from_params(parameters):
+        """
+        Create a download configuration based on some given parameters. Possible parameters are:
+        - anon_download: whether the download is anonymous or not
+        - safe_seeding: whether seeding of the torrent is anonymous
+        - destination: the destination path of the torrent (where it is saved on disk)
+        """
+        download_config = DownloadStartupConfig()
+
+        has_anon_download = 'anon_download' in parameters and len(parameters['anon_download']) > 0
+        has_safe_seeding = 'safe_seeding' in parameters and len(parameters['safe_seeding']) > 0
```

Comment:
(Laurens): If now someone specifies (anon_download = false), then your code will think it's true. This is more a flag based approach than an argument based approach.